### PR TITLE
Update upgrade docs and some download links

### PR DESCRIPTION
**Pull request by serenashe**
Signed-off-by: serenashe <serenashe@kubesphere.io>

/assign @Patrick-LuoYu

### CODE REVIEW

File: content/en/docs/devops-user-guide/examples/create-multi-cluster-pipeline.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ See the table below for the role of each cluster.
 
 {{< notice note >}}
 
-These Kubernetes clusters can be hosted across different cloud providers and their Kubernetes versions can also vary. Recommended Kubernetes versions for KubeSphere v3.1.0: v1.17.9, v1.18.8, v1.19.8 and v1.20.4.
+These Kubernetes clusters can be hosted across different cloud providers and their Kubernetes versions can also vary. Recommended Kubernetes versions for KubeSphere v3.2.0: v1.19.x, v1.20.x, v1.21.5, and v1.22.1.
```
[the remark this answers is not in the document]
Comment (serenashe) [reply content/en/docs/devops-user-guide/examples/create-multi-cluster-pipeline.md:43]: Something went wrong when I tried to push another commit in this PR. I think maybe we could add this in later PRs? 👀

[the remark this answers is not in the document]
Comment (Felixnoo) [reply content/en/docs/devops-user-guide/examples/create-multi-cluster-pipeline.md:43]: Sure.

**Pull request merged**